crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -548,14 +559,16 @@ module deepbook::clob {
                     emit_order_canceled<BaseAsset, QuoteAsset>(pool_id, maker_order);
                 } else {
                     let filled_base_quantity =
-                        if (taker_base_quantity_remaining >= maker_base_quantity) { maker_base_quantity }
+                        if (taker_base_quantity_remaining > maker_base_quantity) { maker_base_quantity }
```

Comment:
This looks like a bugfix--can we please add it in a separate PR with a test that exercises the bug?